### PR TITLE
refactor: use Time::now()

### PR DESCRIPTION
**Pull request by kenjis**
- use Time::now()
- refactor tests

### CODE REVIEW

File: tests/Controllers/LoginTest.php [hunk 1]
```diff
@@ -93,6 +93,8 @@ public function testLoginActionEmailSuccess(): void
 
         // Session should have `logged_in` value with user's id
         $this->assertSame($this->user->id, session('user')['id']);
+
+        Time::setTestNow();
     }
 
     public function testAfterLoggedInNotDesplayLoginPage(): void
```
Comment (datamweb) [inline tests/Controllers/LoginTest.php:100]:
```suggestion
    public function testAfterLoggedInNotDisplayLoginPage(): void
```

Comment (datamweb) [reply tests/Controllers/LoginTest.php:100]: @kenjis this is my misspelling, please correct it if possible.

Comment (kenjis) [reply tests/Controllers/LoginTest.php:100]: Okay. I'll do it.

Comment (kenjis) [reply tests/Controllers/LoginTest.php]: Done.

**Review by MGatner — APPROVED** [review]

**Pull request merged**